site/content/en/guides/consumer/compositepkgs/_index.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Get and apply a remote composite package"
+linkTitle: "Composite Package"
+weight: 7
+type: docs
+description: >
+  Fetch a composite package from a remote git repository and apply its contents to
+  a cluster
+---
+
+{{% pageinfo color="warning" %}}
+
+#### Notice: Composite packages feature support is in alpha phase
+
+{{% /pageinfo %}}
+
+_A kpt package is a directory of resource configs with valid Kptfile in it.
+A composite package is a kpt package with 1 or more subpackages in it._
+
+## Steps
+
+1. [Fetch a remote package](#fetch-a-remote-package)
+2. [View the package contents](#view-the-package-contents)
+3. [Set the setter parameters](#set-the-setter-parameters)
+
+## Fetch a remote package
+
+Fetch an example package which has nested subpackages using [kpt pkg get].
+
+### get command
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/hello-composite-pkg \
+hello-composite-pkg
+```
+
+## View the package contents
+
+The primary package artifacts are Kubernetes resource configuration
+(e.g. YAML files).
+
+### tree command
+
+```sh
+kpt cfg tree hello-composite-pkg/
+```
+
+Output:
+
+```sh
+hello-composite-pkg
+├── [Kptfile]  Kptfile hello-composite-pkg
+├── [deploy.yaml]  Deployment default/hello-composite
+└── hello-subpkg
+    ├── [Kptfile]  Kptfile hello-subpkg
+    ├── [deploy.yaml]  Deployment default/hello-sub
+    ├── hello-subpkg/hello-dir
+    │   └── [configmap.yaml]  ConfigMap default/hello-cm
+    └── hello-subpkg/hello-nestedpkg
+        ├── [Kptfile]  Kptfile hello-nestedpkg
+        └── [deploy.yaml]  Deployment default/hello-nested
+```
+
+There are three kpt packages in the output:
+
+1. hello-composite-pkg
+2. hello-subpkg
+3. hello-nestedpkg
+
+hello-dir is a subdirectory of hello-subpkg and is not a kpt package, as it doesn't
+have a Kptfile in it. Each kpt package is an independent building block and
+should contain resources(ex: setter definitions) of it's own.
+
+### count command
```

Comment:
You're just listing commands here without an explanation. A user guide should have a flow, and well...guide the user.

Reply:
I added more information and a flow to achieve a task. PTAL.